mappings/net/minecraft/item/ModelTransformationMode.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_byhguelf net/minecraft/item/ModelTransformationMode
```

Comment:
Can we move this class to `net/minecraft/client/render/model/json`?

Reply:
one concern is that I used the package to signify that it's unique to items, but I like that refactor